src/pages/main-page/LandingSection.tsx

```diff
@@ -13,6 +15,45 @@ export default function LandingSection(props: any) {
                 <img src={mainBackgroundOverlay} className={styles.overlay}/>
                 <IconCircle/>
             </div>
+            <div className={styles.sectionWrapper}>
+                <div className={styles.sectionTitle}>
+                    {__('landingSection.title')}
+                </div>
+                <div className={styles.sectionDescription}>
+                    {__('landingSection.description')}
+                </div>
+                <div className={styles.sectionButtons}>
+                    <Link to={'/projects'} className={classes(styles.sectionButton, styles.sectionProjectButton)}>
+                        {__('landingSection.buttons.projects')}
+                    </Link>
+                    <Link to={'/sponsors'} className={classes(styles.sectionButton, styles.sectionSponsorButton)}>
+                        {__('landingSection.buttons.sponsors')}
+                    </Link>
+                </div>
+                <div className={styles.sectionNumberContainer}>
+                    <div className={classes(styles.sectionMemberNumber, styles.sectionNumberTitle)}>
+                        <div className={styles.sectionNumber}>
+                            60
+                        </div>
+                        {__('landingSection.numbers.members')}
+                    </div>
+                    <div className={classes(styles.sectionYearNumber, styles.sectionNumberTitle)}>
+                        <div className={styles.sectionNumber}>
+                            2017
+                        </div>
+                        {__('landingSection.numbers.year')}
+                    </div>
+                    <div className={classes(styles.sectionStartNumber, styles.sectionNumberTitle)}>
+                        <div className={styles.sectionNumber}>
```

Comment:
maybe extract to component? e.g.


`<div className={classes(styles.sectionNumberTitle, ...(props.customClasses ?? []))}>`
                        `<div className={styles.sectionNumber}>`
                            {props.number}
                      ` </div>`
                        {props.text}
 `</div>`
(above code could be incorrect, just example)